python/ray/includes/ray_config.pxd

```diff
@@ -0,0 +1,83 @@
+from libc.stdint cimport int64_t, uint64_t
+from libcpp.string cimport string as c_string
+from libcpp.unordered_map cimport unordered_map
+
+
+cdef extern from "ray/ray_config.h" nogil:
```

Comment:
Is the plan to simply keep this in sync with `ray_config.h`? If so, can we put a prominent note near the top of `ray_config.h` saying 

```
IF YOU MODIFY THIS FILE and add a configuration parameter, you must change
at least three additional things:
    1. You must update the "initialize" method in the "RayConfig" class.
    2. You must update the file "ray/python/ray/includes/ray_config.pxd".
    3. You must update the file "ray/python/ray/includes/ray_config.pxi".
```

Reply:
done